### PR TITLE
MOC 10 new planes

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/inys_haen.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Inys Haen
+ManaCost:no cost
+Types:Plane Cridhe
+T:Mode$ PlaneswalkedTo | ValidCard$ Card.Self | Execute$ TrigMill | TriggerZones$ Command | TriggerDescription$ When you planeswalk to CARDNAME and at the beginning of your upkeep, mill three cards.
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigMill | TriggerZones$ Command | Secondary$ True | TriggerDescription$ When you planeswalk to CARDNAME and at the beginning of your upkeep, mill three cards.
+SVar:TrigMill:DB$ Mill | NumCards$ 3 | Defined$ You
+T:Mode$ PlaneswalkedFrom | ValidCard$ Plane.Self | Execute$ TrigChangeZoneAll | TriggerDescription$ When you planeswalk away from CARDNAME, each player returns all land cards from their graveyard to the battlefield tapped
+SVar:TrigChangeZoneAll:DB$ ChangeZoneAll | Origin$ Graveyard | Destination$ Battlefield | ChangeType$ Land
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/inys_haen.txt:8]: Tapped missing

File: forge-gui/res/cardsfolder/upcoming/ketria.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigCounter | TriggerZones$ Command | Secondary$ True | TriggerDescription$ When you planeswalk to CARDNAME and at the beginning of your upkeep, put your choice of vigilance, menace, or trample counter on target creature you control.
+SVar:TrigCounter:DB$ PutCounter | ValidTgts$ Creature.YouCtrl | CounterType$ Vigilance,Menace,Trample | CounterNum$ 1 | CounterTypePerDefined$ True
+T:Mode$ PlanarDice | Result$ Chaos | TriggerZones$ Command | Execute$ RolledChaos | TriggerDescription$ Whenever chaos ensues, exile cards from the top of your library until you exile a nonland permanent card. Put that card onto the battlefield or into your hand.
+SVar:RolledChaos:DB$ DigUntil | ValidPlayer$ You | Valid$ Permanent.nonLand | ValidDescription$ nonland permanent | FoundDestination$ Exile | RevealedDestination$ Exile | RememberFound$ True | SubAbility$ DBChoose
+SVar:DBChoose:DB$ GenericChoice | Choices$ Battlefield,Hand | Defined$ You
+SVar:Battlefield:DB$ ChangeZone | Defined$ Remembered | Origin$ Exile | Destination$ Battlefield | SubAbility$ DBCleanup | SpellDescription$ Put the nonland permanent onto the battlefield
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ketria.txt:10]: Use `DestinationAlternative$`

File: forge-gui/res/cardsfolder/upcoming/littjara.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ManaCost:no cost
+Types:Plane Kaldheim
+T:Mode$ PlaneswalkedTo | ValidCard$ Card.Self | Execute$ TrigToken | TriggerZones$ Command | TriggerDescription$ When you planeswalk to CARDNAME and at the beginning of your upkeep, create a 2/2 blue Shapeshifter creature token with changeling. (It is every creature type.)
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigToken | TriggerZones$ Command | Secondary$ True | TriggerDescription$ When you planeswalk to CARDNAME and at the beginning of your upkeep, create a 2/2 blue Shapeshifter creature token with changeling. (It is every creature type.)
+SVar:TrigToken:DB$ Token | TokenScript$ u_2_2_shapeshifter_changeling | TokenOwner$ You
+T:Mode$ PlanarDice | Result$ Chaos | TriggerZones$ Command | Execute$ RolledChaos | TriggerDescription$ Whenever chaos ensues, choose a creature type. Put a +1/+1 counter on each creature you control of that type.
+SVar:RolledChaos:DB$ ChooseType | Defined$ You | Type$ Creature | SubAbility$ DBPutCounterAll | AILogic$ MostProminentComputerControls
+SVar:DBPutCounterAll:DB$ PutCounterAll | ValidCards$ Creature.ChosenType+YouCtrl | CounterType$ P1P1 | CounterNum$ 1 | Zone$ Battlefield
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/littjara.txt:9]: Remove non existing param Zone

File: forge-gui/res/cardsfolder/upcoming/paliano.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Paliano
+ManaCost:no cost
+Types:Plane Fiora
+T:Mode$ DamageDone | ValidSource$ Creature.YouCtrl | ValidTarget$ Player | CheckSVar$ Monarch | SVarCompare$ EQ0 | CombatDamage$ True | Execute$ TrigMonarch | TriggerZones$ Command | TriggerDescription$ When one or more creatures you control deal combat damage to a player, if there is no monarch, you become the monarch.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/paliano.txt:4]: wrong trigger (one or more)

File: forge-gui/res/cardsfolder/upcoming/strixhaven.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Strixhaven
+ManaCost:no cost
+Types:Plane Arcavios
+S:Mode$ Continuous | AddKeyword$ Demonstrate | EffectZone$ Command | Affected$ Instant,Sorcery | AffectedZone$ Stack | Description$ Instant and sorcery spells players cast have demonstrate. (Whenever a player casts an instant or sorcery spell, they may copy it. If they do, they choose an opponent to also copy it. Players may choose new targets for their copies.)
+T:Mode$ PlanarDice | Result$ Chaos | TriggerZones$ Command | Execute$ RolledChaos | TriggerDescription$ Whenever chaos ensues, return up to one target instant or sorcery card from a graveyard to its owner's hand
+SVar:RolledChaos:DB$ ChangeZone | Origin$ Graveyard | TargetMin$ 0 | TargetMax$ 1 | Destination$ Hand | ValidTgts$ Creature | TgtPrompt$ Select up to target instant or sorcery spell from any graveyard
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/strixhaven.txt:6]: Wrong target

File: forge-gui/res/cardsfolder/upcoming/the_caldaia.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Name:The Caldaia
+ManaCost:no cost
+Types:Plane Capenna
+S:Mode$ Continuous | EffectZone$ Command | Affected$ Creature.YouCtrl | AffectedZone$ Stack | AddKeyword$ Blitz:CardManaCost:Spell.Creature+wasCastFromYourHand | Description$ Creature spells you cast from your hand have blitz {3}. (If you cast a spell for its blitz cost, it gains haste and "When this creature dies, draw a card." Sacrifice it at the beginning of the next end step.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_caldaia.txt:4]: wrong cost

File: forge-gui/res/cardsfolder/upcoming/the_wilds.txt [hunk 7]
```diff
@@ -0,0 +1,13 @@
+Name:The Wilds
+ManaCost:no cost
+Types:Plane Eldraine
+T:Mode$ PlaneswalkedTo | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When you planeswalk to CARDNAME and at the beginning of your upkeep, create a Food token.
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigToken | TriggerZones$ Command | Secondary$ True | TriggerDescription$ When you planeswalk to CARDNAME and at the beginning of your upkeep, create a Food token.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_food_sac | TokenOwner$ You
+T:Mode$ PlanarDice | Result$ Chaos | TriggerZones$ Command | Execute$ RolledChaos | TriggerDescription$ Whenever chaos ensues, target player sacrifices a creature. If they do, you create a Food token. You create two Food tokens instead if the sacrificed creature's toughness was 4 or greater.
+SVar:RolledChaos:DB$ Sacrifice | ValidTgts$ Player | SacValid$ Creature | SacMessage$ Creature | SubAbility$ DBToken
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_wilds.txt:8]: "if they do" not handled

**Comment by Simisays** [comment]
@tool4ever Is this PR good to go or is something still missing

**Comment by tool4ever** [comment]
Sorry, I'm traveling for a while so I try to sneak in some time for reviews or small fixes whenever I can but tbh yours usually take a few rounds so I'll cycle back to them eventually if no one is able to beat me to it

**Comment by Simisays** [comment]
@tool4ever Oh perfectly fine, take all the time you need. I was just wondering whether "Chaos Ensures" needed to be changed first before these planes could be added but it makes sense now

**Review by tool4ever — APPROVED** [review]

**Pull request merged**